app/views/users/couches/new.html.haml

```diff
@@ -0,0 +1,28 @@
+%h1 Add Couch
```

Comment:
I'll make a card to add some html structure to this. Thanks for laying the groundwork!